misc/horizontal-scale-file-create.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+N="$1"
+if [ -z "$N" ]; then
+    N=1
+fi
+
+function do_scale_test {
+    T=$(mkdir /cephfs-$N)
+    cd "$T"
+    for ((i = 0; i < 100; ++i)); do
+      mkdir "$i"
+      while j <= 9000; do
```

Comment:
O.o

Does this while loop actually work? Why not a for loop?

Reply:
Oops I thought I pushed it. The for and the while should be terminated by "done".

Reply:
Okay, ill change it to for loop